### PR TITLE
helm charts: Apply the resource requests/limits customization

### PR DESCRIPTION
**Pull request by haoxins**

### CODE REVIEW

File: helm-chart/flink-operator/values.yaml [hunk 1]
```diff
@@ -39,8 +39,8 @@ securityContext:
 
 resources:
   requests:
-    cpu: 100m
-    memory: 128Mi
+    cpu: 500m
+    memory: 512Mi
```
Comment (regadas) [inline helm-chart/flink-operator/values.yaml:43]: Hi @haoxins what were you seeing with the previous amount of resource requests?

Comment (haoxins) [reply helm-chart/flink-operator/values.yaml:43]:
```zsh
$ k top pod flink-session-cluster-jobmanager-0 --containers
POD                                  NAME         CPU(cores)   MEMORY(bytes)
flink-session-cluster-jobmanager-0   jobmanager   8m           396Mi
$ k top pod flink-session-cluster-taskmanager-0 --containers
POD                                   NAME          CPU(cores)   MEMORY(bytes)
flink-session-cluster-taskmanager-0   sidecar       0m           0Mi
flink-session-cluster-taskmanager-0   taskmanager   11m          574Mi
```
without any workload

Comment (regadas) [reply helm-chart/flink-operator/values.yaml]: hm, these resources should be related to the operator itself; Also, after a closer look I believe they are not even being applied 😞

Comment (haoxins) [reply helm-chart/flink-operator/values.yaml]: > Also, after a closer look I believe they are not even being applied 

I think it's useful to support custom `requests/limits`, want me to apply theses?

Comment (regadas) [reply helm-chart/flink-operator/values.yaml]: Yeah, if you have time that would be great! I already created an issue for it https://github.com/spotify/flink-on-k8s-operator/issues/259

Comment (haoxins) [reply helm-chart/flink-operator/values.yaml]: @regadas I just applied the Values and changed the PR's title.

**Review by regadas — APPROVED** [review]
Awesome @haoxins Thanks for doing this!

**Pull request merged**